overlord/ifacestate/ifacestate.go

```diff
@@ -195,6 +195,11 @@ func connect(st *state.State, change *state.Change, installedSnapTask *state.Tas
 	connectPlugConnection := hookstate.HookTask(st, summary, connectPlugHookSetup, initialContext)
 	connectPlugConnection.WaitFor(connectSlotConnection)
 
+	// store IDs of the connect-plug/slot- hook tasks in the main 'connect' task,
```

Comment:
This looks unrelated to the PR

Reply:
Indeed, thanks again for spotting, this is something I removed in parent PR (which already landed) and now erronously brought it back when de-conflicting.